strapi/strapi
# fix[documentation] changed id type to integer && updated [ru] translation

### What does it do?

Just changed the id type from 'number' to 'integer'.

### Why is it needed?

Actually, it's wrong to think about id like a `number`, because it has more concrete type - `integer`. 
You should use `number` in openapi spec only in cases when you don't know for sure which format of `number` it could be (ex: float, decimal, integer). 
But in case of Strapi: we completely sure that id field is `integer` and it cannot be anything else.

For example, openapi-codegen generates `*float32` in Golang for id field when it has `number` type.... and it's correct behaviour, so we need to fix this typing issue here.

### How to test it?

Add documentation plugin and trigger openapi spec generation.
Then you will see that all `id` fields have type `integer`
